### ISSUE TITLE
White navbar icon is too light (Photon Dark theme)

### ISSUE THREAD

**Issue by fvsch** [mozilla/testpilot-containers#722]
Using Nightly with the built-in Dark theme, icons in the navbar appear light gray. The Containers icon appears pure white, which can be a bit jarring.

<img width="182" alt="Screenshot" src="https://user-images.githubusercontent.com/243601/29212838-cb398dd8-7ea0-11e7-9d7a-90d2358b0fc4.png">

Here’s how Photon is applying this color:

1. Using a SVG image with `list-style-image`. The SVG uses `fill="context-fill"` (which is a proprietary addition in Gecko).
For instance the "Developer" icon is `chrome://browser/skin/developer.svg` with this code:
```svg
<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
  <path fill="context-fill" d="M14.555 3.2l-2.434 2.436a1.243 1.243 0 1 1-1.757-1.757L12.8 1.445A3.956 3.956 0 0 0 11 1a3.976 3.976 0 0 0-3.434 6.02l-6.273 6.273a1 1 0 1 0 1.414 1.414L8.98 8.434A3.96 3.96 0 0 0 11 9a4 4 0 0 0 4-4 3.956 3.956 0 0 0-.445-1.8z"/>
</svg>
```

2. The context color is the toolbar’s text color, `rgba(249, 249, 250, .7)` in this context.

Note that`fill=context-fill` works in Firefox 55+.
https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-context-properties

A low-tech solution might be to change the PNG images to use a hardcoded pure white at 70% opacity, that would be close enough.

**Comment by emmyemi** [comment]
I think this _should_ be fixed when #710 gets fixed.

**Comment by jonathanKingston** [comment]
It also depended on a patch I worked on, which reminds me I need to uplift. https://bugzilla.mozilla.org/show_bug.cgi?id=1388907

Thanks for raising, will dupe this.
